lms/envs/aws.py

```diff
@@ -883,14 +883,8 @@
 CREDIT_HELP_LINK_URL = ENV_TOKENS.get('CREDIT_HELP_LINK_URL', CREDIT_HELP_LINK_URL)
 
 #### JWT configuration ####
-DEFAULT_JWT_ISSUER = ENV_TOKENS.get('DEFAULT_JWT_ISSUER', DEFAULT_JWT_ISSUER)
```

Comment:
I assume we can retire all the ENV_TOKENS uses from configuration, edx-internal and someone from devops can look in the secure repos for it?
Can you clean out the config/internal repos?

Reply:
Yes, I have open PRs for cleaning up those configs: 
* https://github.com/edx/edx-internal/pull/504
* https://github.com/edx/configuration/pull/4706

Reply:
Also, I am keeping a list of Authn related things that we will probably need/want to clean up as we enable Split Frontends: https://openedx.atlassian.net/browse/ARCH-204